test/parallel/test-fs-open-flags.js

```diff
@@ -37,20 +37,27 @@ assert.strictEqual(stringToFlags('xa+'), O_APPEND | O_CREAT | O_RDWR | O_EXCL);
 ('+ +a +r +w rw wa war raw r++ a++ w++ x +x x+ rx rx+ wxx wax xwx xxx')
   .split(' ')
   .forEach(function(flags) {
-    assert.throws(function() { stringToFlags(flags); });
+    assert.throws(
+      () => stringToFlags(flags),
```

Comment:
I think if you do it that way, then the linter requires the long regexp to be aligned with the first argument, resulting in violating the no-more-than-80-chars-in-a-line rule. If that's the case, I'm OK with the way it is here. (Actually, I'm OK with it regardless, but I don't feel strongly about it if someone else thinks having the first argument on the same line as the opening parenthesis is more readable.)